pkgs/by-name/li/libcgroup/package.nix

```diff
@@ -21,15 +23,30 @@ stdenv.mkDerivation rec {
     hash = "sha256-kWW9ID/eYZH0O/Ge8pf3Cso4yu644R5EiQFYfZMcizs=";
   };
 
+  configureFlags =
+    [
+      (lib.enableFeature enableSystemd "systemd")
+    ]
+    # implicit declaration of function 'rpl_malloc', ; did you mean 'realloc'
+    #
+    # It looks like in case of cross-compilation, autoconf assumes that malloc of the
```

Comment:
Shouldn't this be a broader check that also catches other cross-compilation methods?